### ISSUE TITLE
eigs call requires Arpack package

### ISSUE THREAD

**Issue by AmebaBrain** [davidavdav/GaussianMixtures.jl#39]
Function call below requires imported `Arpack` package
https://github.com/davidavdav/GaussianMixtures.jl/blob/bf2b7f48f57b3e2c74a953861de3a1cffe2e187d/src/train.jl#L173

> Start creating model...
> ┌ Info: 0: avll = 
> └   tll[1] = -0.5383099222506905
> ERROR: LoadError: UndefVarError: eigs not defined
> Stacktrace:
>  [1] gmmsplit(::Array{Float64,1}, ::Array{Float64,2}, ::Float64) at /home/toleg/.julia/packages/GaussianMixtures/RGtTJ/src/train.jl:173

After adding `using Arpack` to the head of train.jl it starts to work with warning 
> ┌ Warning: Package GaussianMixtures does not have Arpack in its dependencies:
> │ - If you have GaussianMixtures checked out for development and have
> │   added Arpack as a dependency but haven't updated your primary
> │   environment's manifest file, try `Pkg.resolve()`.
> │ - Otherwise you may need to report an issue with GaussianMixtures
> └ Loading Arpack into GaussianMixtures from project dependency, future warnings for GaussianMixtures are suppressed.

I added `Arpack` to REQUIRE and was trying to call `Pkg.resolve()` but warning didn't dissapear

I can raise PR with this single addition but because of warning I think this is not the only thing that should be done

**Comment by davidavdav** [comment]
Yes, they changed the Packaging system, and although it has become better, I still have to battle it each time.  I can't find an upgrade manual for going from the old system to the new system. 

These seem to have been steps I needed to take:
```julia
] dev ../GaussianMixtures.jl
] instantiate .
] activate .
] add <a whole bunch of packages>
```
I added the packages manually by trial and error, `] resolve` seems not to resolve the dependencies of this package.
